templates/repo/diff/box.tmpl

```diff
@@ -90,13 +90,13 @@
 						<div class="diff-file-box diff-box file-content {{TabSizeClass $.Editorconfig $file.Name}} gt-mt-3" id="diff-{{$file.NameHash}}" data-old-filename="{{$file.OldName}}" data-new-filename="{{$file.Name}}" {{if or ($file.ShouldBeHidden) (not $isExpandable)}}data-folded="true"{{end}}>
 							<h4 class="diff-file-header sticky-2nd-row ui top attached normal header gt-df gt-ac gt-sb">
 								<div class="diff-file-name gt-df gt-ac gt-mr-3">
-									<a role="button" class="fold-file muted gt-mr-2" {{if not $isExpandable}}style="visibility: hidden"{{end}}>
+									<button class="fold-file ui button button-ghost gt-p-0 gt-mr-3" {{if not $isExpandable}}style="visibility: hidden"{{end}}>
```

Comment:
Can replace these inline styles with `gt-invisible` class.

Reply:
Yes, but is that a task for this PR?

Reply:
It's the only case of `visibility: hidden` in all templates, and you are already modifying the line, so I'd go for it 😆

Reply:
Done